scheduler/test/cook/test/mesos/group.clj

```diff
@@ -85,6 +85,14 @@
             job-straggler-b (create-dummy-job conn :group group-ent-id)
             straggler-b (create-dummy-instance conn job-straggler-b :instance-status :instance.status/running
                                              :start-time (tc/to-date (t/ago (t/minutes 121))))
+            ;; Not treated as a straggler because instance already failed as a straggler
```

Comment:
I think that is effectively what is happening because job-straggler-b is detected as a straggler while job-straggler-c is configured to already have a straggler instance so the current instance is not killed.